### ISSUE TITLE
Cannot install Catboost on Docker image build failed on AWS Codebuild

### ISSUE THREAD

**Issue by CodeDeveloperV** [catboost/catboost#2189]
Problem:
I am trying to install catboost when building my docker image in AWS Codebuild.

This is my Dockerfile

```
FROM python:3

RUN pip3 install --upgrade cython
RUN pip3 install --upgrade pip

RUN pip3 install pandas==1.3.5
RUN pip3 install scikit-learn==0.24.2
RUN pip3 install catboost==1.0.5

COPY requirements.txt /

RUN pip3 install -r /requirements.txt

RUN pip3 install --upgrade numpy

COPY . /app
WORKDIR /app
ENTRYPOINT ["./gunicorn.sh"]
```

When trying to do it I get the following error.

```
Step 8/14 : RUN pip3 install catboost==1.0.5
--
197 | ---> Running in 99012e51d38f
198 | ERROR: Could not find a version that satisfies the requirement catboost==1.0.5 (from versions: none)
199 | ERROR: No matching distribution found for catboost==1.0.5
200 | The command '/bin/sh -c pip3 install catboost==1.0.5' returned a non-zero code: 1

```

catboost version:
1.0.5 and latest

Operating System:
I running on AWS CodeBuild - Debian

```

Step 2/14 : RUN cat /etc/os-release
--
107 | ---> Running in f78849199b25
108 | PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
109 | NAME="Debian GNU/Linux"
110 | VERSION_ID="11"
111 | VERSION="11 (bullseye)"
112 | VERSION_CODENAME=bullseye
113 | ID=debian

```

This error does not allow me to install the package or finish the build

**Comment by ryanshrott** [comment]
Any update on this? Facing the same issue.

**Comment by andrey-khropov** [comment]
@CodeDeveloperV 

Sorry that it took us so long to attend to this issue. 
What is the CPU architecture of you try to run this on? I suspect that the issue might be that you ran it on architecture that has not been supported by prebuilt wheels on PyPI at the time like `arm64v8`. Linux aarch64 wheels has been available on PyPI only since release 1.2.

**Comment by andrey-khropov** [comment]
No feedback, so I'm closing this issue. The likely cause of the problem is described [above](https://github.com/catboost/catboost/issues/2189#issuecomment-1548703795). Feel free to reopen if this issue still persist.
